packages/fetchai/skills/aries_faber/handlers.py

```diff
@@ -58,9 +61,22 @@ def __init__(self, **kwargs):
         """Initialize the handler."""
         super().__init__(**kwargs)
 
+        # ACA stuff
+        self.aca_identity = FABER_ACA_IDENTITY
```

Comment:
```suggestion
        self.aca_identity = kwargs.pop("faber_aca_identity", FABER_ACA_IDENTITY)
```
This makes it configurable via yaml. file

Reply:
Done